example/Code_Fernand_Rhizoscan.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # RhizoScan Pipeline
+
+# This pipeline is a step by step pipeline
+
+# ## Common import
+
+# In[ ]:
+
+
+from __future__ import absolute_import
+
+
+# In[ ]:
+
+
+get_ipython().magic(u'pylab notebook')
+from matplotlib import pyplot as plt
+from IPython.display import Image
+ 
+
+
+# ## RhizoScan Import
+
+# In[ ]:
+
+
+from rhizoscan import get_data_path
+from rhizoscan.root.pipeline import load_image, detect_petri_plate, compute_graph, compute_tree
+from rhizoscan.root.pipeline.arabidopsis import segment_image, detect_leaves, _detect_leaves
+from rhizoscan.root.graph.mtg import tree_to_mtg
+from rhizoscan.root.image.seed import _cluster_seed
+
+
+# ## RSA Image
+
+# In[ ]:
+
+
+image_filename ='C:/Users/FORTFER/Desktop/Dossier donnees analysees/Seq 8_Boite 00002_filtre.jpg'
```

Comment:
Add the image on github.
And refer to it locally. Otherelse, the script will not run.